euler.rosinstall

```diff
@@ -7,3 +7,6 @@
     version: euler_integration}
 - git: {local-name: sicktoolbox_wrapper, uri: 'https://github.com/swri-robotics/sicktoolbox_wrapper.git',
     version: euler_integration}
+- git: {local-name: navigation, uri: 'https://github.com/DLu/navigation.git',
```

Comment:
@DLu, do you mind if we fork your repo to swri-robotics.  This way you won't have to worry about/remember to keep that branch around.  Also, do you plan on rolling in "our" changes into the main navigation repo (It's OK if not).